pkgs/by-name/li/libnss_nis/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  autoreconfHook,
+  libxcrypt,
+  libnsl,
+  libtirpc,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libnss_nis";
+  version = "3.2";
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+  ];
+
+  buildInputs = [
+    libxcrypt
+    libnsl
+    libtirpc
+  ];
+
+  src = fetchFromGitHub {
+    owner = "thkukuk";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-dt5wL+v98Heg6395BOwNssXLXmoOKFnRXGqlOknYYPs=";
+  };
+
+  outputs = [ "out" ];
+
+  meta = {
+    description = "NSS module for glibc, to provide NIS support for glibc.";
+    changelog = "https://github.com/thkukuk/${pname}/blob/master/NEWS";
+    homepage = "https://github.com/thkukuk/${pname}";
+    license = lib.licenses.lgpl21;
```

Comment:
```suggestion
    license = lib.licenses.lgpl21Plus;
```

Reply:
I am not really an expert with these legal things, but the [license file](https://github.com/thkukuk/libnss_nis/blob/master/COPYING) of the repo states LGPL 2.1.
Why do you suggest changing that to LGPL 2.1+?

Reply:
The license header in each source file states that later versions are also applicable

Reply:
Oh, I didn't look that close at the license headers.

I have adjusted the license attribute.